terraform/modules/acr/output.tf

```diff
@@ -1,17 +1,7 @@
-output "acr_name" {
-  value = azurerm_container_registry.acr.name
+output "acr_id" {
+  value = azurerm_container_registry.acr.id
 }
 
 output "acr_login_server" {
   value = azurerm_container_registry.acr.login_server
-}
-
-output "acr_admin_username" {
```

Comment:
I'll need to test if the Edge VM can also make use of MI for authenticating to ACR in the private setup (currently I'm still using username/password for pulling the edgeAgent image) - so we might need to re-introduce these variables later but that should be fine.

Reply:
@machteldbogels Did you mean Edge VM using MI to authenticate to ACR for the IoT Edge modules in the deployment manifest?

Reply:
@syedhassaanahmed yes exactly. so far it seems that adding a role assignment for the VM and then removing username/password does not allow the pull automagically. according to these [docs](https://aka.ms/acr/authorization) I'll need to use the az cli to ensure the system identity is being used, but we wouldn't want to install this on the vm (I assume)

Reply:
Got it. In that case I'll revert back to outputting the ACR username and password as managed identity won't work on a real edge device anyway.

Reply:
Done